### DISCUSSION TITLE
Disable OCP Analytics

### DISCUSSION THREAD

**Discussion by WerdaFarooq** [rhubarbgroup/.github#37] [Support (ROC)]
I am currently facing a configuration challenge related to the use of Object Cache Pro in our managed service. We have a specific need to disable the analytics functionality, and we've tried adding the following configuration to the wp-config.php file:

php
Copy code
define('WP_REDIS_CONFIG', [
    'analytics' => [
        'enabled' => false,
        'footnote' => false,
    ],
]);
However, due to the automated nature of our setup, this configuration gets overridden periodically. **Please advise how we can simply get rid of analytics from Object Cache Pro.**

I've explored the possibility of using a separate configuration file and including it after the Redis configuration, but as the WP_REDIS_CONFIG is defined as a constant, it poses a challenge.

If there's a recommended approach or filter hook for achieving this, I would greatly appreciate your guidance.

**Comment by tillkruss** [comment]
Who are you hosting with?
